code/game/objects/items/devices/radio/headset.dm

```diff
@@ -294,32 +294,14 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	name = "Clowning Headset"
 	desc = "A headset with a clowning filter, honk!"
 	icon_state = "honk_headset"
-	var/honken = FALSE  // honk filter
-
-/obj/item/radio/headset/clown/ui_data(mob/user)
-	. = ..()
-	.["honk"] = honken
-
-/obj/item/radio/headset/clown/ui_act(action, params, datum/tgui/ui)
-	. = ..()
-	if(action == "honk")
-		honken = !honken
-		. = TRUE
+	honken = TRUE  // honk filter
+	use_honken = TRUE
```

Comment:
this should start off like other command headsets.

Reply:
oh are those disabled roundstart? i havent played a head in ages